Assets/ConsentManagementProvider/Scripts/model/common/GdprConsent.cs

```diff
@@ -31,8 +31,8 @@ public string ToFullString()
             if(TCData != null)
             {
                 sb.AppendLine("TCData:");
-//                 foreach (var kvp in TCData)
-//                     sb.AppendLine($"    {kvp.Key}: {kvp.Value}");
+                 foreach (var kvp in TCData)
+                     sb.AppendLine($"    {kvp.Key}: {kvp.Value.ToString()}");
```

Comment:
Do we _have_ to call `Value.ToString()`? Some values inside `TCData` may not be a string (sometimes they have booleans like 0 or 1).

Reply:
@andresilveirah `.ToString()` in c# can handle value types just right. 
For example, it will convert a boolean value to string `True` or `False` and a `1` integer will become "1" string.
We can decide not to specify `.ToString()` here but it will be done anyway under the hood.

Reply:
The `TCData.Value` has an `object` type, which means it can be any value-type value that secures the usage of `.ToString()`